charts/application/templates/k8s-rolebinding.yaml

```
@@ -0,0 +1,17 @@
+{{- if or (eq .Values.k8sAPI.role "edit") (eq .Values.k8sAPI.role "view") (eq .Values.k8sAPI.role "admin") }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  labels:
+    app.kubernetes.io/name: {{ $.Release.Name }}
+  name: {{ $.Release.Name }}
```

is the namespace missing here?
```suggestion
  name: {{ $.Release.Name }}
  namespace: {{ .Release.Namespace }}
```